R/experiments.R

```diff
@@ -184,13 +184,28 @@ get_experiment <- function(experiment_id = get_active_experiment_id(), experimen
     new_mlflow_experiment()
 }
 
+#' Get Experiment ID
+#'
+#' Makes a call to `get_experiment` and returns just the ID.
+#'
+#' @param experiment_name The experiment name. This field is required.
+#' @inheritParams create_experiment
+#'
+#' @export
+get_experiment_id <- function(experiment_name, client = mlflow_client()) {
+  get_experiment(
+    experiment_name = experiment_name,
+    client = client
+  )$experiment_id
```

Comment:
i don't think we need to check for whether something is returned from `get_experiment` because it will just error in there, preventing a bad `$experiment_id` pluck if nothing is returned?

Reply:
Yeah what happens if you supply an `experiment_name` that doesn't exist? I'd expect the API to throw an error

Reply:
> get_experiment_id("foo")
 Error: MLFlow API call failed with status code 404.

Diagnostics:
RESOURCE_DOES_NOT_EXIST
Could not find experiment with name 'foo'

Reply:
so yes, the error is thrown in `get_experiment`

Reply:
Okay yeah, that seems good to me then. I'd just leave it as is